src/main/java/software/amazon/smithy/gradle/SmithyPlugin.java

```diff
@@ -107,19 +94,19 @@ private void addCliDependencies(Project project) {
         DependencySet existing = cli.getAllDependencies();
 
         if (existing.stream().anyMatch(d -> isMatchingDependency(d, "smithy-cli"))) {
-            project.getLogger().info("Using explicitly configured Smithy CLI");
+            LOGGER.info("Using explicitly configured Smithy CLI");
```

Comment:
All Smithy-specific logging statements are now handled by JUL. This makes it much easier to find Smithy logging statements separate from Gradle's.